wordview/text_analysis/core.py

```diff
@@ -268,6 +290,45 @@ def do_txt_analysis(
 
     global ftmodel
 
+    tags_list = [
```

Comment:
Since you allow the user to provide a list of target tags in the calling class i.e. `wrapper.TextStatsPlots` via the `pos_tags` parameter, best to pass that same param to `tags_list` so that we don't run the analysis for the POS tags that are not desired by the user.

Reply:
In the latest commit, I'm now passing the pos_tags as a parameter to do_txt_analysis.